### PR TITLE
Feature/i18n

### PR DESCRIPTION
**Pull request by Cactu5**
Adds missing translations and English translations.
Add ability to change language.
Fix tests.

### CODE REVIEW

**Comment by Cactu5** [comment]
Some tests need to be fixed since localization will change some data that are checked in tests.

File: src/main/resources/i18n/Messages.properties [hunk 1]
```diff
@@ -12,6 +12,12 @@ loginForm.label.username=Var god och ange ert användarnamn, tack:
 loginForm.label.password=Var god och ange ert lösenord, tack:
 
 loginForm.text.dontHaveAnAccount=Åh nej, vännen. Saknar ni ett konto? Snälla var så snäll och och samt klicka på följande länk:
+loginForm.text.register=Skapa konto
+loginForm.text.login=Logga in
+
+loginForm.success.logout= Hurra vännen! Utloggningen lyckades!
+
+loginForm.error.invalidLogin =Åh nej, vännen. Inloggningen misslyckades.
```
Comment (william-stacken) [inline src/main/resources/i18n/Messages.properties:20]: Alltså dessa meddelanden. 🤣

Comment (Cactu5) [reply src/main/resources/i18n/Messages.properties:20]: 😂

**Comment by william-stacken** [comment]
LGTM!

**Review by william-stacken — APPROVED** [review]
Here's the approval!

**Pull request merged**